04/css/mobile.css

```diff
@@ -0,0 +1,50 @@
+.nav__menu {
+  display: none;
+  margin: 0 auto;
+  width: fit-content;
+  text-align: left;
+}
+
+.dropdown__link,
+.nav__link {
+  font-size: 1.5rem;
+  color: var(--dropdown-color);
+}
+
+input:checked + .nav__menu {
+  display: block;
+}
+
+input {
+  display: none;
+}
+
+.dropdown {
+  display: block;
+  padding-left: 1rem;
+}
+
+.submenu {
+  display: block;
+  padding-left: 1rem;
+}
+
+/* hamburger icon  */
+.nav__menu__hamburger {
```

Comment:
Jeśli mamy BEM to nie możemy w nieskończoność robić _ ... _ ... _ - taki zapis występuje między Blockiem, a Elementem tj. `block-name__element-name`. Możesz utworzyć nowy Block o nazwie `hamburger`, nowy element o nazwie `nav__hamburger` lub `manu__hamburger` lub modyfikator o nazwie `nav__menu--hamburger` ale ten ostatni musiałby występować razem z elementem `nav__menu`